src/Forum/Controller/SavePasswordController.php

```diff
@@ -52,6 +59,8 @@ public function handle(Request $request)
         $password = array_get($input, 'password');
         $confirmation = array_get($input, 'password_confirmation');
 
+        $this->validator->assertValid(compact('password'));
+
         if (! $password || $password !== $confirmation) {
```

Comment:
@franzliedke I think we can make use of validator for checking password confirmation so it can throw localised error as in Laravel